src/undo.h

```diff
@@ -34,9 +34,9 @@ struct TxInUndoFormatter
 
     template<typename Stream>
     void Unser(Stream &s, Coin& txout) {
-        unsigned int nCode = 0;
+        uint32_t nCode = 0;
         ::Unserialize(s, VARINT(nCode));
-        txout.nHeight = nCode / 2;
+        txout.nHeight = nCode >> 1;
```

Comment:
Why this change?

Reply:
> In the same spirit, I also changed the nCode / 2 into nCode >> 1 to match undo.h and coins.h.

To match the code in `undo.h` and `coins.h` (as it is done with `fCoinBase`). Can remove it if people prefer ?

Reply:
🤷‍♂ don't make a case for me.

Reply:
Hmm ok so.. Waiting for a second person opinion to remove it ?

Reply:
Agree with @promag.

Reply:
This pull request has 5 ACKs. Let's please not invalidate all the review and fight about style changes at this point.

Reply:
Yeah, I wasn't trying to invalidate existing reviews, I also gave my ACK. I was trying to understand the reason behind this change. I think it's good practice to not include unrelated changes.

Reply:
I think it is fine to do style-changes in code that reviewers have to look at anyway. It could have been better if it was in different commits or in the same commit, but with explanation in the commit message why each change is made.
Let's just keep it in mind for the future.

Reply:
My bad - I just though the change was related enough to be included here. I'll do this in a separate commit in the future. Thanks.